### PR TITLE
new cask cappuccino v3.0,275

### PR DESCRIPTION
**Pull request by yurikoles**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask-versions/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask-versions/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/cappuccino-beta.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cask 'cappuccino-beta' do
+  version '3.0,275'
+  sha256 '9b91bfd49717de41ab0e24492f2379d99f65c0c30ec5563b8af2c0f0640d9393'
+
+  # rink.hockeyapp.net/api was verified as official when first introduced to the cask
+  url "https://rink.hockeyapp.net/api/2/apps/902c8f3a570242808ebb00495f5fa04d/app_versions/#{version.after_comma}?format=zip"
```
Comment (vitorgalvao) [inline Casks/cappuccino-beta.rb]: Can’t find URL on the site.

Comment (yurikoles) [reply Casks/cappuccino-beta.rb]: second result in google://"cappuccino beta"

Comment (vitorgalvao) [reply Casks/cappuccino-beta.rb]: > second result in google://"cappuccino beta"

For future reference, Google changes results depending on who does the search, so a direct link is always better.

We’ll need something official, though. Something on the official website that links to hockeyapp.

**Comment by reitermarkus** [comment]
> in commit comment

Which comment?

**Comment by yurikoles** [comment]
This cask is now available as `brew cask install yurikoles/yurikoles/cappuccino-beta`

**Pull request closed without merging**